packages/data/addon/services/keg.js

```diff
@@ -77,21 +78,22 @@ export default class KegService extends Service {
     const factory = this._getFactoryForType(options.modelFactory || type);
     const recordKeg = this._getRecordKegForType(type);
     const idIndex = this._getIdIndexForType(type);
+    const namespace = options.namespace || getDefaultDataSourceName();
```

Comment:
Instead of teaching the key about data sources (right now it is a general service), can we have a generic default namespace name and gave the keg clients do the defaulting to the `getDefaultDataSourceName()`